browser/brave_screenshots/test/brave_screenshots_unittests.cc

```diff
@@ -0,0 +1,138 @@
+      content::ContextMenuParams params,
+      bool is_pwa_browser = false) {
+    auto menu = std::make_unique<BraveRenderViewContextMenuMock>(
+        *web_contents->GetPrimaryMainFrame(), params);
+    Browser::CreateParams create_params(
+        is_pwa_browser ? Browser::Type::TYPE_APP : Browser::Type::TYPE_NORMAL,
+        profile_.get(), true);
+    auto test_window = std::make_unique<TestBrowserWindow>();
+    create_params.window = test_window.get();
+    browser_.reset(Browser::Create(create_params));
+    menu->SetBrowser(browser_.get());
+    menu->Init();
+    return menu;
+  }
+
+  void SetUp() override {
+    TestingProfile::Builder builder;
+    profile_ = builder.Build();
+    web_contents_ = content::WebContents::Create(
+        content::WebContents::CreateParams(profile_.get()));
+  }
+
+  void SetBraveScreenshotsFeatureState(bool enabled) {
+    features_.Reset();
+    features_.InitWithFeatureState(
+        brave_screenshots::features::kBraveScreenshots, enabled);
+  }
+
+  void TearDown() override {
+    web_contents_.reset();
+    browser_.reset();
+    profile_.reset();
+
+    // We run into a DCHECK on Windows. The scenario is addressed explicitly
+    // in Chromium's source for MessageWindow::WindowClass::~WindowClass().
+    // See base/win/message_window.cc for more information.
+    ui::Clipboard::DestroyClipboardForCurrentThread();
+  }
+
+  PrefService* GetPrefs() { return profile_->GetPrefs(); }
+
+ private:
+  // NOLINTNEXTLINE (clang-tidy readability-identifier-naming)
+  content::BrowserTaskEnvironment browser_task_environment;
+  ScopedTestingLocalState testing_local_state_;
+  base::test::ScopedFeatureList features_;
+  std::unique_ptr<TestingProfile> profile_;
+  std::unique_ptr<Browser> browser_;
+  std::unique_ptr<content::WebContents> web_contents_;
+};
+
+// We expect screenshot menu items to be present only when enabled
+TEST_F(BraveScreenshotsContextMenuTest, MenuForWebPage) {
+  content::ContextMenuParams params;
+  params.page_url = GURL("https://example.com");
+
+  for (auto enabled : {true, false}) {
+    SetBraveScreenshotsFeatureState(enabled);
+    auto context_menu = CreateContextMenu(GetWebContents(), params);
+    EXPECT_TRUE(context_menu);
+
+    // Check for the main submenu label
+    std::optional<size_t> index =
+        context_menu->menu_model().GetIndexOfCommandId(
+            IDC_BRAVE_SCREENSHOT_TOOLS);
+
+    if (enabled) {
+      EXPECT_TRUE(index.has_value());
+      EXPECT_TRUE(context_menu->menu_model().GetSubmenuModelAt(*index));
+    } else {
+      EXPECT_FALSE(index.has_value());
+    }
+  }
+}
+
+// We expect all menu items to be absent within developer tools' context menu
+TEST_F(BraveScreenshotsContextMenuTest, MenuForDevTools) {
+  SetBraveScreenshotsFeatureState(true);
+
+  content::ContextMenuParams params;
+  params.page_url = GURL("devtools://devtools");
+
+  auto context_menu = CreateContextMenu(GetWebContents(), params, true);
```

Comment:
why is `is_pwa_browser` true in this case?

Reply:
Oversight. I believe I sourced this test originally from the context menu tests, and forgot to remove that.